XamlControlsGallery/ControlPages/ItemsRepeaterPage.xaml.cs

```diff
@@ -402,7 +421,32 @@ private void UpdateSortAndFilter()
             var peer = FrameworkElementAutomationPeer.FromElement(VariedImageSizeRepeater);
 
             peer.RaiseNotificationEvent(AutomationNotificationKind.Other, AutomationNotificationProcessing.ImportantMostRecent, $"Filtered recipes, {sortedFilteredTypes.Count()} results.", "RecipesFilteredNotificationActivityId");
-        }
+        }
+
+        private void OnAnimatedScrollRepeaterKeyDown(object sender, Windows.UI.Xaml.Input.KeyRoutedEventArgs e)
+        {
+            if (e.Handled != true)
+            {
+                var targetIndex = -1;
+                if (e.Key == Windows.System.VirtualKey.Home)

```

Comment:
Are these the only keys we need to handle here?  What about PgUp/PgDown, for example?

Reply:
PageUp and PageDown are handled by the wrapping ScrollViewer. This is not a full fledged implementation of a collection scenario for sure.